systers_portal/static/css/style.css

```diff
@@ -354,6 +354,19 @@ table.decoration-none  tr a {
   margin-right: 15px;
 }
 
+.text-bottom{
+    padding: 10px;
+    border-bottom: 1px solid #eee;
+}
+
+.upcomingMeetupslist{
+  height: 400px;
+  overflow-y: scroll;
+  line-height: 1em;
+  word-wrap: break-word;
```

Comment:
What do you see if you use `overflow-wrap` in place of `word-wrap`?

Reply:
Both results in the same, like how it looks in the screenshot. "overflow-wrap: word-wrap property has been renamed overflow-wrap in the current draft of the CSS3 Text specification" Should I modify it to overflow-wrap?